1.12.0/zfs-operator-centos7-0.9.0-RC2.yaml

```diff
@@ -1333,8 +1333,9 @@ apiVersion: apps/v1
 metadata:
   name: openebs-zfs-controller
   namespace: kube-system
-  openebs.io/component-name: openebs-zfs-controller
-  openebs.io/version: 0.9.0-RC2
+  labels:
```

Comment:
@pawanpraka1  The labels was incorrect in the RC2 yaml. I am not sure whether this was intentionally added, since its not present in the RC1 tag.

Reply:
yes, first time we are introducing the zfs-localpv to openebs/charts. I had copied the yaml from the repo and added RC1 tag in the yaml, don't know who added RC2 yaml. Did not know that we need the label. What is the purpose of this label?

Reply:
> What is the purpose of this label?

Its added to all the openebs components that are deployed. Basically to identify the version of component deployed.